content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -17,11 +17,16 @@ Support for views is currently a very early preview feature. You can add a view
 
 Database views allow you to name and store queries. In relational databases, views are [stored SQL queries](https://www.postgresql.org/docs/current/sql-createview.html) that might include columns in multiple tables, or calculated values such as aggregates. In MongoDB, views are queryable objects where the contents are defined by an [aggregation pipeline](https://www.mongodb.com/docs/manual/core/aggregation-pipeline) on other collections.
 
-This guide explains how to enable the `views` preview feature and add views to your Prisma schema.
+The `views` preview feature allows you to represent views in your Prisma schema with the `view` keyword. To use views in Prisma, follow these steps:
+
+- [Enable the `views` preview feature](#enable-the-views-preview-feature)
+- [Create a view in the underlying database](#create-a-view-in-the-underlying-database)
+- [Add the view to your Prisma schema](#add-views-to-your-prisma-schema)
+- [Query the view in Prisma Client](#query-views-in-prisma-client)
```

Comment:
I would add the step to add the view in a migration file via `--create-only`

Reply:
Isn't that covered by "Create a view in the underlying database"? It just does not use Migrate but manually - but that is where this would happen.

Reply:
Added this as part of rewriting the steps